packages/fcl/src/fcl.test.js

```diff
@@ -1,3 +1,6 @@
+// mock send function so getChainId does not hit the chain at startup
+config.put("sdk.transport", async ix => ix)
```

Comment:
@huyndo Can we confirm that this has no ability to affect `config` used in other tests?

If we need this for this FCL test file, we could use `config.overload` as a way to isolate the configuration change to a specific test(s):

eg:
```javascript
config.overload({ "sdk.transport": ix => ix }, async () => {
   ... do test stuff...
})
```

Reply:
@JeffreyDoyle Welp turns out not adding to resolver pipeline means I can get rid of this too